keep/common/management/commands/update_pidman_title.py

```diff
@@ -0,0 +1,266 @@
+
+            """
+            sys.stdout.write(dry_run_notice)
+            self.is_dry_run = True
+
+        # verify environments
+        environment_notice = """
+        Please confirm that you would like to proceed with the
+        following environments:
+        FEDORA: %s
+        PIDMAN: %s
+        """ % (settings.FEDORA_ROOT, settings.PIDMAN_DOMAIN)
+        self.stdout.write(environment_notice)
+
+        # confirm to proceed
+        confirmation = raw_input("Verify above information and type 'yes' to proceed: ")
+        if (confirmation != "yes"):
+            self.stdout.write("Answer is not 'yes'. Operation aborted.")
+            sys.exit(1)
+
+        # setup output directory paths
+        self.current_time_string = time.strftime("%Y%m%d-%H%M%S", time.localtime())
+        self.output_path = "%s/%s/%s/" % (os.getcwd(), "tmp", self.current_time_string)
+        self.error_path = "%s/errors/" % self.output_path
+
+        # create output directories
+        if not os.path.exists(self.output_path):
+            os.makedirs(self.output_path)
+        if not os.path.exists(self.error_path):
+            os.makedirs(self.error_path)
+
+        # create log files
+        self.summary_log = open(("%s/%s" % (self.output_path, "summary.csv")), "w+")
+
+        # start the object collection process
+        self.stdout.write("Started collecting objects from Fedora...")
+
+        # get objects of different content models
+        audio_objects = self.repo.get_objects_with_cmodel(AudioObject.AUDIO_CONTENT_MODEL)
+        self.stderr.write("%i Audio objects found." % len(audio_objects))
+
+        video_objects = self.repo.get_objects_with_cmodel(Video.VIDEO_CONTENT_MODEL)
+        self.stderr.write("%i Video objects found." % len(video_objects))
+
+        arrangement_objects = self.repo.get_objects_with_cmodel(ArrangementObject.ARRANGEMENT_CONTENT_MODEL)
+        self.stderr.write("%i Arrangement objects found." % len(arrangement_objects))
+
+        diskimage_objects = self.repo.get_objects_with_cmodel(DiskImage.DISKIMAGE_CONTENT_MODEL)
+        self.stderr.write("%i DiskImage objects found." % len(diskimage_objects))
+
+        simple_collection_objects = self.repo.get_objects_with_cmodel(SimpleCollection.COLLECTION_CONTENT_MODEL)
+        self.stderr.write("%i SimpleCollection objects found." % len(simple_collection_objects))
+
+        collection_objects = self.repo.get_objects_with_cmodel(CollectionObject.COLLECTION_CONTENT_MODEL)
+        self.stderr.write("%i Collection objects found." % len(collection_objects))
+
+        mailbox_objects = self.repo.get_objects_with_cmodel(boda.Mailbox.MAILBOX_CONTENT_MODEL)
+        self.stderr.write("%i Mailbox objects found." % len(mailbox_objects))
+
+        # end the object collection process and start
+        self.stdout.write("Object collection finished.")
+
+        # update objects in each content model
+        self.update_progress(audio_objects, "AudioObject", self.pidman)
```

Comment:
Why pass in the pidman client to the method? Why not make it accessible on the class?